packages/plugin-infra/src/types.ts

```diff
@@ -305,14 +305,36 @@ export namespace Plugin.SNSAdaptor {
     // #endregion
 
     export interface ApplicationEntry {
+        ID: string
         /**
          * Render entry component
          */
-        RenderEntryComponent: (props: { disabled: boolean }) => JSX.Element
+        RenderEntryComponent?: (props: {
+            disabled: boolean
+            icon: React.ReactNode
```

Comment:
It should bring its own icon and title instead of reading it from the props. It's an unnecessary pass.

Reply:
done